assignment/README.md

```diff
@@ -55,6 +70,8 @@ are hard to use and think about. Global variables are a strategy for
 coping with that difficulty by breaking data out of the asynchronous
 context.
 
+**The program will require a lot of different options based on user input. Making sure that we coordinate this cleanly and effectively will be the biggest hurdle we're thinking of.**
```

Comment:
I will often try to sketch this kind of thing out on paper. Basically, I imagine the flow of data (inputs -> outputs) and try to draw up a wire diagram that I can refer back to when I'm writing code